doc/policy_tutorial/POLICY_RUN_LOCALLY.md

```diff
@@ -92,6 +99,7 @@ $ curl localhost:8080
 </body>
 </html>
 ```
+>**NOTE**: alernatively you can get the docker container ip address and curl from your local machine `APICAST_IP=$(docker inspect apicast_build_0-development-1 | yq e -P '.[0].NetworkSettings.Networks.apicast_build_0_default.IPAddress' -)` and `curl -i -H "Host: one" "http://${APICAST_IP}:8080/?user_key=0123456789"`
```

Comment:
```suggestion
> **NOTE:**
Alternatively, you can get the docker container IP address and curl from your local machine:
> - `APICAST_IP=$(docker inspect apicast_build_0-development-1 | yq e -P '.[0].NetworkSettings.Networks.apicast_build_0_default.IPAddress' -)` 
> - `curl -i -H "Host: one" "http://${APICAST_IP}:8080/?user_key=0123456789"`
```

Reply:
Fixed the capitalization and spelling of `alernatively` and attempting to make the note into a block with the commands more clear in bullets.

Reply:
suggestion accepted